graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -232,6 +264,9 @@ def _rename_name_and_add_to_record(self, node):
             return node
 
         new_name_string = self.renamings.get(name_string, name_string)  # Default use original
+        if new_name_string is None:
+            # suppress the type
+            return REMOVE
```

Comment:
With this additional line of code, type-hinting this function is really tricky because now you can return a node or `REMOVE`. Here, `REMOVE` is `None` but that could change so just saying something like `Optional[Node]` doesn't necessarily work.

Reply:
Good catch! Let's ignore this for now and type it as `Optional[Node]`, and leave a comment explaining the above in the code. That way, if `mypy` starts complaining one day, we'll know that the type hint may need to change to match the new value of `REMOVE`.